src/Makefile.am

```diff
@@ -890,7 +890,7 @@ bitcoin_util_LDADD = \
 
 # bitcoin-chainstate binary #
 bitcoin_chainstate_SOURCES = bitcoin-chainstate.cpp
-bitcoin_chainstate_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES) $(BOOST_CPPFLAGS)
+bitcoin_chainstate_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
```

Comment:
FWIW this won't have any effect on c-i currently because boost is installed into the common depends prefix.

As a quick hack to demonstrate the functionality of this PR, you could add: https://github.com/theuni/bitcoin/commit/e131d8f1e33baa9a9499d2e1a4a99b171566c5a5 and drop it before merge. That puts boost in a different path, so `BOOST_CPPFLAGS` becomes a requirement as-intended. (We should also fix that for good in a less hackish way.)

As far as I can tell this PR should currently fail with the above patch because the removal is not yet done as mentioned here: https://github.com/bitcoin/bitcoin/pull/28335/commits/bdbc5093c226a8f843f938cc51e2077c4b4623a0#r1313092304

Reply:
> As a quick hack to demonstrate the functionality of this PR, you could add: [theuni@e131d8f](https://github.com/theuni/bitcoin/commit/e131d8f1e33baa9a9499d2e1a4a99b171566c5a5) and drop it before merge. That puts boost in a different path, so `BOOST_CPPFLAGS` becomes a requirement as-intended.

IIRC, this issue is recurring, no?

> We should also fix that for good in a less hackish way.

Agree with it :)

Reply:
Heh, yes, I think this is the 3rd time we've needed this to demonstrate a boost dependency removal. Will do.